cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -147,9 +147,11 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
     }
     section("Role Management", "administration-security-roles", "enterprise-edition") {
       initQueries("CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED",
-        "CREATE USER user1 SET PASSWORD 'abc'", "CREATE USER user2 SET PASSWORD 'abc'", "CREATE USER user3 SET PASSWORD 'abc'",
-        "CREATE ROLE role1", "CREATE ROLE role2")
+        "CREATE USER user1 SET PASSWORD 'abc'", "CREATE USER user2 SET PASSWORD 'abc'", "CREATE USER user3 SET PASSWORD 'abc'")
       p("Roles can be created and managed using a set of Cypher administration commands executed against the `system` database.")
+      important {
```

Comment:
I'm not sure about using an `IMPORTANT` admonition  - the other admonitions on this page are `NOTE` (see line 124).

Reply:
I was torn between those two. Well, if you are voting for `NOTE` I'll make it so